.github/workflows/query-engine-black-box.yml

```diff
@@ -17,27 +17,28 @@ concurrency:
 
 jobs:
   rust-tests:
-    name: "Test query-engine as a black-box"
+    name: 'Test query-engine as a black-box'
 
     strategy:
       fail-fast: false
       matrix:
         database:
-          - name: "postgres15"
+          - name: 'postgres15'
             single_threaded: false
-            connector: "postgres"
-            version: "15"        
+            connector: 'postgres'
+            # Why are we testing v15 and not older?
+            version: '15'
```

Comment:
👀

Reply:
I asked in Slack, let's see https://prisma-company.slack.com/archives/C4GCG53BP/p1696585370897559